src/resources/views/crud/fields/select.blade.php

```diff
@@ -2,6 +2,7 @@
 @php
 	$current_value = old(square_brackets_to_dots($field['name'])) ?? $field['value'] ?? $field['default'] ?? '';
     $entity_model = $crud->getRelationModel($field['entity'],  - 1);
+    $field['allows_null'] = $field['allows_null'] ?? $crud->model::isColumnNullable($field['name']);
```

Comment:
```suggestion
    $field['allows_null'] ??= $crud->model::isColumnNullable($field['name']);
```

I don't know if we should adopt this or not, it's a shortcut, **I like and prefer it**, but one may say it's less readable.
@tabacitu do you want to give your feedback on this?

Reply:
I think that `??=` shorthand was introcuced only in PHP 7.4 https://wiki.php.net/rfc/null_coalesce_equal_operator.

Reply:
Sure 😬, nevermind! 😅